darwin/bin/build_pack.sh

```diff
@@ -18,6 +18,28 @@ rm -R Icon.iconset
 
 ignoring="(bin|test|Procfile)"
 
+if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
```

Comment:
This `if` is not necessary because all the darwin scripts are only run when `$TRAVIS_OS_NAME` is `osx` (see line 45 and 46 of `.travis.yml`).
It's better to not have Travis variables in this file in case you need to run the build on a local machine, the script doesn't depend on a Travis variable, only the `.travis.yml` should have those variables.